.circleci/unittest/linux/scripts/run_test.sh

```diff
@@ -21,6 +21,6 @@ export CKPT_BACKEND=torch
 
 python .circleci/unittest/helpers/coverage_run_parallel.py -m pytest test/smoke_test.py -v --durations 200
 python .circleci/unittest/helpers/coverage_run_parallel.py -m pytest test/smoke_test_deps.py -v --durations 200 -k 'test_gym or test_dm_control_pixels or test_dm_control or test_tb'
-python .circleci/unittest/helpers/coverage_run_parallel.py -m pytest --instafail -v --durations 200 --ignore test/test_distributed.py --ignore test/test_rlhf.py
+python .circleci/unittest/helpers/coverage_run_parallel.py -m pytest --instafail -v --durations 200
```

Comment:
(this file isn't used anymore I believe)
test_rlhf should still be excluded

Reply:
Yeah we can probably just delete this file in a follow-up since it's not used anymore.